packages/core/strapi/src/commands/utils/logger.ts

```diff
@@ -1,5 +1,6 @@
 import chalk from 'chalk';
 import ora from 'ora';
+import * as cliProgress from 'cli-progress';
```

Comment:
just to check, we verified this is supporting windows terms ?

Reply:
cli-progress is designed for linux/macOS/container applications which mostly providing standard compliant tty terminals/shells. In non-tty mode it is suitable to be used with logging daemons (cyclic output).

It also works with PowerShell on Windows 10 - the legacy command prompt on outdated Windows versions won't work as expected and is not supported!

Reply:
ok worth QAing on windows then 👍

Reply:
QA on windows passed:
https://strapihq.slack.com/archives/C06MG4B728G/p1717664941898799